tools/python/mwm_feature_compare.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import print_function
+
+import argparse
+import os
+import subprocess
+import multiprocessing
+
+OMIM_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..')
+FIND_FEATURE = os.path.join(OMIM_ROOT, 'tools', 'python', 'mwm', 'find_feature.py')
+
+
+def count_feature(mwm_path, feature_name):
+    p = subprocess.Popen(['/usr/bin/python', FIND_FEATURE, mwm_path, 'et', feature_name], stdout=subprocess.PIPE)
+    features_count = sum(1 for _ in p.stdout)
+    return features_count
+
+
+def compare_feature_num(args_tuple):
+    old_mwm, new_mwm, feature_name, threshold = args_tuple
+    old_feature_count = count_feature(old_mwm, feature_name)
+    new_feature_count = count_feature(new_mwm, feature_name)
+
+    if new_feature_count - old_feature_count < 0:
```

Comment:
for readability (considering the next line,
```
if old_feature_count - new_feature_count > 0
```
Or add a variable ```delta```.

Reply:
Fixed, thanks!